execute_cbt.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#This script runs the end-to-end cbt scripts
+#example: ./execute_cbt.sh <host address> <username> <password> <vdi uuid>
+
+HOST=$1
+USER=$2
+PASSWORD=$3
+VDI=$4
+
```

Comment:
Minor - I'd typically just just use the following here:
set -eux

See http://gfxmonk.net/2012/06/17/my-new-bash-script-prelude.html.